pages/contribute/README.md

```diff
@@ -0,0 +1,50 @@
+---
+navigation_title: "Contribute"
+position: 5
+changed: "2018-06-15"
+---
+
+# Contribute to this guide
+
+**This guide is the result of tight collaboration between accessibility experts and web developers. Its content by far is not exhaustive yet: there remain a lot of topics to talk about, and due to the fast evolution of web technologies, there certainly will always be.**
+
+We welcome everyone who wants to join our initiative, be it with adding new contents, fixing typos or bugs, etc.
+
+## What you can do
```

Comment:
I don't think we need this heading; it's the only one on level 2.

Reply:
Agree